src/Plugin/search_api/backend/SearchApiElasticsearchBackend.php

```diff
@@ -43,6 +43,11 @@ class SearchApiElasticsearchBackend extends BackendPluginBase implements PluginF
   use PluginFormTrait;
 
   /**
+   * Set a large integer to be the size for a "Hard limit" value of "No limit".
+   */
+  const FACET_NO_LIMIT_SIZE = 10000;
```

Comment:
Is 10000 facets really needed? I wonder if a limit of 1000 makes more sense.

Reply:
@deviantintegral I think it is OK because this constant is more like the case we don't have any limit but by default Elasticsearch limits it to 10 items, so 10k sounds more like unlimited.